### PR TITLE
nobody likes titlecase

### PR DESCRIPTION
**Pull request by romdotdog**

### CODE REVIEW

File: src/Compiler/Errors/Errors.ts [hunk 1]
```diff
@@ -38,40 +38,40 @@ export const enum BriskErrorType {
 // Error Strings
 export const BriskErrorMessage = {
   // Brisk Error
-  [BriskErrorType.CompilerError]: 'Compiler Bug Please Report',
-  [BriskErrorType.FeatureNotYetImplemented]: 'Feature Not Yet Implemented',
-  [BriskErrorType.ImportNotSupported]: 'Your Current Environment Does Not Support Imports',
+  [BriskErrorType.CompilerError]: 'A compiler bug has occurred',
+  [BriskErrorType.FeatureNotYetImplemented]: 'Not implemented',
```
Comment (spotandjake) [inline src/Compiler/Errors/Errors.ts:42]: Feature not yet implemented

File: src/Compiler/Errors/Errors.ts [hunk 2]
```diff
@@ -38,40 +38,40 @@ export const enum BriskErrorType {
...
+  [BriskErrorType.ImportNotSupported]: 'Your environment does not support imports',
   // Syntax Error
-  [BriskErrorType.CannotUseReservedKeyword]: 'Cannot Use Reserved Keyword %1',
+  [BriskErrorType.CannotUseReservedKeyword]: 'Cannot use reserved keyword `%1`',
   [BriskErrorType.ReturnStatementsOnlyValidInsideFunction]:
-    'Return Statements Only Valid Inside Function',
+    'Return statements must be inside a function',
   [BriskErrorType.InvalidNumber]: 'Number `%1` Is Invalid',
```
Comment (spotandjake) [inline src/Compiler/Errors/Errors.ts:48]: Missed One

File: src/Compiler/Errors/Errors.ts [hunk 3]
```diff
@@ -38,40 +38,40 @@ export const enum BriskErrorType {
...
-  [BriskErrorType.FlagExpectedArguments]: 'Flag %1 Expected %2 Arguments, Found %3',
-  [BriskErrorType.ConstantAssignment]: 'Unexpected Assignment To Constant Variable %1',
-  [BriskErrorType.DuplicateField]: 'Duplicate Field %1',
-  [BriskErrorType.DuplicateExport]: 'Duplicate Export %1',
-  [BriskErrorType.ModuleDoesNotExist]: 'Module %1 Does Not Exist',
-  [BriskErrorType.InvalidExport]: 'Invalid Export %1',
-  [BriskErrorType.InvalidTypeName]: 'Invalid Type Name %1',
-  [BriskErrorType.TypeHasAlreadyBeenDeclared]: 'Type %1 Has Already Been Declared',
-  [BriskErrorType.TypeNotFound]: 'Type %1 Not Found',
+  [BriskErrorType.InvalidFlag]: 'Decorator `%1` does not exist', // TODO: rename
```
Comment (spotandjake) [inline src/Compiler/Errors/Errors.ts:51]: Flag

**Review by spotandjake — CHANGES_REQUESTED** [review]

File: src/Compiler/Errors/Errors.ts (hunk 3 above)
Comment (romdotdog) [reply src/Compiler/Errors/Errors.ts:51]: No.

File: src/Compiler/Errors/Errors.ts (hunk 2 above)
Comment (romdotdog) [reply src/Compiler/Errors/Errors.ts:48]: What is an invalid number ?????? This error is not even used anywhere ??????

File: src/Compiler/Errors/Errors.ts (hunk 3 above)
Comment (spotandjake) [reply src/Compiler/Errors/Errors.ts:51]: Flag

**Review by spotandjake — CHANGES_REQUESTED** [review]

File: src/Compiler/Errors/Errors.ts (hunk 3 above)
Comment (romdotdog) [reply src/Compiler/Errors/Errors.ts:51]: Seriously?

Comment (spotandjake) [reply src/Compiler/Errors/Errors.ts:51]: Flag

**Review by spotandjake — CHANGES_REQUESTED** [review]

File: src/Compiler/Errors/Errors.ts (hunk 3 above)
Comment (romdotdog) [reply src/Compiler/Errors/Errors.ts:51]: Why??

Comment (spotandjake) [reply src/Compiler/Errors/Errors.ts:51]: Flag

File: src/Compiler/Errors/Errors.ts [hunk 4]
```diff
@@ -38,40 +38,41 @@ export const enum BriskErrorType {
...
-  [BriskErrorType.FlagExpectedArguments]: 'Flag %1 Expected %2 Arguments, Found %3',
-  [BriskErrorType.ConstantAssignment]: 'Unexpected Assignment To Constant Variable %1',
-  [BriskErrorType.DuplicateField]: 'Duplicate Field %1',
-  [BriskErrorType.DuplicateExport]: 'Duplicate Export %1',
-  [BriskErrorType.ModuleDoesNotExist]: 'Module %1 Does Not Exist',
-  [BriskErrorType.InvalidExport]: 'Invalid Export %1',
-  [BriskErrorType.InvalidTypeName]: 'Invalid Type Name %1',
-  [BriskErrorType.TypeHasAlreadyBeenDeclared]: 'Type %1 Has Already Been Declared',
-  [BriskErrorType.TypeNotFound]: 'Type %1 Not Found',
+  [BriskErrorType.InvalidFlag]: 'Flag `%1` does not exist', // TODO: rename
```
Comment (spotandjake) [inline src/Compiler/Errors/Errors.ts]: Remove TODO.

File: src/Compiler/Errors/Errors.ts [hunk 5]
```diff
@@ -38,40 +38,41 @@ export const enum BriskErrorType {
...
-  [BriskErrorType.InvalidArgumentLength]: 'Function Expects A Maximum Of %1 Arguments, Found %2',
-  [BriskErrorType.WasmExpressionUnknown]: 'Wasm Expression %1 Could Not be Found',
-  [BriskErrorType.DeadCode]: 'Found Dead Code',
+    'Optional parameters must appear last in parameter list',
+  [BriskErrorType.TypeMisMatch]: 'Expected type `%1` found `%2`',
+  [BriskErrorType.IncompatibleTypes]: 'Type `%1` is incompatible with `%2`',
+  [BriskErrorType.TypeCouldNotBeInferred]: 'Type `%1` cannot be inferred',
+  [BriskErrorType.ArrayTypeLengthCouldNotBeInferred]: 'Array type length cannot be inferred',
+  [BriskErrorType.InvalidArgumentLength]: 'Function expects a maximum of %1 arguments, found %2',
+  [BriskErrorType.WasmExpressionUnknown]: 'Instruction `%1` does not exist', // TODO: rename
```
Comment (spotandjake) [inline src/Compiler/Errors/Errors.ts:68]: Add Wasm
Remove TODO

**Review by spotandjake — CHANGES_REQUESTED** [review]

**Comment by romdotdog** [comment]
I can't take it anymore.

**Comment by spotandjake** [comment]
> _ No description provided. _

No Description Provided

**Comment by romdotdog** [comment]
HALLELUJAH

**Pull request merged**